mysql-test/suite/tokudb.rpl/t/rpl_tokudb_row_img_blobs.test

```diff
@@ -0,0 +1,62 @@
+#Want to skip this test from daily Valgrind execution
+--source include/no_valgrind_without_big.inc
+
+#
+# This file contains tests for WL#5096.
+#
+
+-- let $rpl_topology= 1->2->3
+-- source include/rpl_init.inc
+-- source include/have_binlog_format_row.inc
+
+-- connection server_1
+-- source include/have_innodb.inc
```

Comment:
IMHO cleaning up things like that depends on the effort required, in general it's OK to copy existing testcases as-is, but this particular bit is too bizarre for me and it looks like it can be removed and --source include/have_innodb.inc / have_tokudb.inc added to the top of the testcase instead